hydrus/conf.py

```diff
@@ -37,18 +38,36 @@
     # try to load the path specified in environment variable
     apidoc_env = os.environ['APIDOC_REL_PATH']
     apidoc_path = cwd_path / Path(apidoc_env)
+    FOUND_DOC = True
 except KeyError:
     # if it is not set, use the example doc
-    apidoc_path = cwd_path / 'examples' / 'drones' / 'doc.py'
+    FOUND_DOC = False
+    apidoc_path = cwd_path / 'hydrus' / 'samples' / 'hydra_doc_sample.py'
 
 try:
-    APIDOC_OBJ = SourceFileLoader(
-        'doc',
-        str(apidoc_path)).load_module().doc
+    # support jsonld, py, yaml APIDOC formats
+    apidoc_format = str(apidoc_path).split(".")[-1]
+    if apidoc_format == 'jsonld':
```

Comment:
please factorise this condition into a function that takes a path so we can write:

```
APIDOC_OBJ = load_apidoc(path)
```
this module is alaready too verbose, try to make it readable and concise by writing functions whenever some beahviour is apt to be factorised.

Reply:
I have made two functions get_apidoc_path() and load_api_doc(). I think making functions to set other global variables such as PORT, DEBUG, API_NAME and DB_URL will not increase readability. I find them concise right now. What do you suggest?